connection_notifier.hh

```diff
@@ -49,5 +49,3 @@ struct client_data {
 
 future<> notify_new_client(client_data cd);
 future<> notify_disconnected_client(gms::inet_address addr, client_type ct, int port);
-
-future<> clear_clientlist();
```

Comment:
Don't add this at all please.

Reply:
Ultimately the code from `main()` went here, so I kept it